src/compose/images.ts

```diff
@@ -74,13 +84,36 @@ export class Images extends (EventEmitter as {
 		this.docker = opts.docker;
 		this.logger = opts.logger;
 		this.db = opts.db;
+		this.config = opts.config;
+	}
+
+	public async init() {
+		this.appUpdatePollInterval = await this.config.get('appUpdatePollInterval');
```

Comment:
If the appUpdatePollInterval in config changes, it won't be updated here. You can add a `this.config.on('change',fn)` like in DeviceState (where you update maxPollTime), or have a setter that's called from DeviceState? (if you do the latter, you can just have that setter instead of an init function, and no need to have Config passed on the constructor)